O/OpenCV/build_tarballs.jl

```diff
@@ -98,9 +123,9 @@ products = [
 dependencies = [
     Dependency(PackageSpec(name="Qt5Base_jll", uuid="ea2cea3b-5b76-57ae-a6ef-0a8af62496e1"))
     Dependency(PackageSpec(name="Libglvnd_jll", uuid="7e76a0d4-f3c7-5321-8279-8d96eeed0f29"))
-    BuildDependency(PackageSpec(name="libjulia_jll", version=julia_version))
+    BuildDependency(PackageSpec(name="libjulia_jll"))
     Dependency(PackageSpec(name="libcxxwrap_julia_jll"))
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"7")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version = v"9", julia_compat="1.6")
```

Comment:
Is GCC 9 strictly needed?  Also, please don't shuffle arguments around needlessly, that increases noise in the diff
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"9")
```

Reply:
Fixed. Some of the dependencies also lower bound GCC 9